testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/CustomPrefabSpawnerForPerformanceTests.cs

```diff
@@ -28,14 +28,15 @@ public NetworkObject Instantiate(ulong ownerClientId, Vector3 position, Quaterni
             return networkObject;
         }
 
-        public void Destroy(NetworkObject networkObject)
+        public bool Destroy(NetworkObject networkObject)
```

Comment:
Not sure I see the reason for this change as it always returns true?

Reply:
Actually, look at [NetworkPrehabHandler.cs]( https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/955/files/cfa48e63e63298dfef8aeb7a5139ec471536c35f#diff-e22f792a99552567173262df910cf3a252e21b874dd342fa4ddc1f3cd16dc531), the INetworkPrefabInstanceHandler interface. It is part of the requirement for the implementation.  It provides the user with the ability to specify whether they want the NetworkObject to be destroyed or not.  For the unit test, this case is always true.